src/js/video.js

```diff
@@ -147,6 +148,15 @@ videojs.hooks_ = {};
  */
 videojs.hooks = function(type, fn) {
   videojs.hooks_[type] = videojs.hooks_[type] || [];
+  if (fn && type === 'vttjsloaded') {
+    vttjsOnLoad((vttjs) => {
```

Comment:
it can't be outside of vttjsloader because only that module knows if and when vttjs has loaded. Keeping it there, I think, it good because it contains all (except this little bit, but this mostly deals with hooks) the things that work with loading vttjs.